homeassistant/components/opentherm_gw/binary_sensor.py

```diff
@@ -45,20 +45,31 @@ def __init__(self, gw_dev, var, device_class, friendly_name_format):
         self._device_class = device_class
         self._friendly_name = friendly_name_format.format(gw_dev.name)
         self._unsub_updates = None
+        self._unsub_options = None
+        self._force_update = False
```

Comment:
This will cause the setting to revert to the default when Home Assistant restarts until the options flow is followed again.
The setting is available from `config_entry.options` in `async_setup_entry`, so we need to feed it into `__init__()` and use it here if it was set (see `climate.py` for a similar construction).

Reply:
Good catch. I've fixed this in the latest commit and also extracted the default value into a constant in const.py